ci/lint.yml

```diff
@@ -6,7 +6,7 @@ image_resource:
   type: docker-image
   source:
     repository: golang
-    tag: 1.16.4
+    tag: latest
```

Comment:
Should this be a specific version like the others?

Reply:
I think the agreement is to use `latest` for `lint` and `audit`.

https://github.com/ONSdigital/dp-apipoc-server/pull/18#discussion_r711161269
https://github.com/ONSdigital/dp-frontend-articles-controller/pull/1#discussion_r713883761